packages/node-sdk/src/client.ts

```diff
@@ -583,6 +626,12 @@ export class BucketClient {
   }
 
   private _getFeatures(context: Context): Record<string, RawFeature> {
+    void this.syncContext(context);
```

Comment:
```suggestion
if (enableTracking=true) void this.syncContext(context)
```

Reply:
I would prefer to keep it as is due to a log inside, otherwise I'd have to copy-paste the same one in other parts.